rero_ils/templates/rero_ils/admin/page.html

```diff
@@ -21,53 +21,70 @@
   <meta name="google-site-verification" content="{{google_id}}"/>
   {%- endfor %}
   {%- endif %}
+  {%- block head_title %}
+    {%- set title = title or _(config.THEME_SITENAME) or _('Invenio') %}
+    <title>{{title}}</title>
+  {%- endblock head_title %}
+  {%- block head_links %}
+    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}"/>
+    {%- if keywords %}<link rel="canonical" href="{{ canonical_url }}"/>{% endif %}
+    {%- block head_links_langs %}
+      {%- if alternate_urls %}
+        {%- for alt_ln, alternate_url in alternate_urls.items() %}
+    <link rel="alternate" hreflang="{{ alt_ln }}" href="{{ alternate_url }}"/>
+        {%- endfor %}
+      {%- endif %}
+    {%- endblock %}
+    {%- block head_apple_icons %}
```

Comment:
why apple?

Reply:
Because it's the link for the favicon for the apple ecosystem. This part comes from the invenio-theme. https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/templates/invenio_theme/page.html#L39